import-express.html

```diff
@@ -103,9 +103,8 @@ <h3>Page preview</h3>
                                 <sp-theme color="light" scale="medium">
                                     <div id="mapping-editor">
                                         <div class="row">
-                                            <h2 id="sec-color">Color</h2>
-                                            <h2 id="sec-dom-selector">Id/Classes</h2>
-                                            <h2 id="sec-layout">Layout<br />(cols x rows)</h2>
```

Comment:
I think we'll need `cols and rows` later (did you try www.hoyalover.com -> 3x3 image grid near the bottom?).
Seems like you want it out though, so I guess we can add it back in later if it comes up.